hasher-matcher-actioner/hmalib/common/writebacker_models.py

```diff
@@ -180,8 +193,8 @@ def _writeback_impl(self, writeback_signal: BankedSignal) -> str:
             "review_status": "REVIEWED_MANUALLY",
             "status": "MALICIOUS",
         }
-        print(descriptors)
 
+        # If we already have a descriptor we can copy it and upload make sure to never expire it
```

Comment:
same nits as before: `s\copy\update` and `s\make sure to never expire it\set it to not expire`

Reply:
I like "copy" here because the function is `copy_threat_descriptor`. Reworded otherwise

Reply:
again nit but because it has come up twice, 'copy' is a fault in the function name the comment is trying to correct not reinforce...
The call outs I think we want to address here or in a docstring because it is overly confusing are:
1) That this will just result in 1 descriptor even though the word copy is in the method name
2) This descriptor is set to not expire but could be later if something changes (that is why I was against the word 'never')
@jeberl